test/wdio/shadow-dom-basic/cmp.test.tsx

```diff
@@ -0,0 +1,65 @@
+import { h } from '@stencil/core';
+import { render } from '@wdio/browser-runner/stencil';
+
+describe('shadow-dom-basic', () => {
+  beforeEach(() => {
+    render({
+      template: () => <shadow-dom-basic-root></shadow-dom-basic-root>,
+    });
+  });
+
+  it('render', async () => {
+    await $('shadow-dom-basic-root').waitForExist();
+    const elm = document.querySelector('shadow-dom-basic-root');
+
+    if ('attachShadow' in HTMLElement.prototype) {
+      await testNativeShadow(elm);
```

Comment:
I migrated a similar test and just got rid of the polyfilled version since today we expect Shadow DOM support when using Stencil, right?

Reply:
Yep, removing the polyfilled tests now